canonical-web-and-design/ubuntu.com
# Build new /smartstart page

## Done

- Add new page `/smartstart`
(confirmed with @SirSamTumless create new page at `/smartstart` and don't add to `/smart-start)`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/smartstart
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1H8JymT13-pHvPh2KXeXYaAH4HoPYWiwAQXXUDqlGzLM/edit#heading=h.wwhog9r2zrb1)
- Using design similar to [#3178](https://github.com/canonical-web-and-design/web-squad/issues/3178)


## Issue / Card

Fixes [#3367](https://github.com/canonical-web-and-design/web-squad/issues/3367)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/97700803-53557400-1aa4-11eb-9def-9d6c20f5820e.png)
